src/azure-cli/azure/cli/command_modules/network/tests/latest/test_private_endpoint_commands.py

```diff
@@ -326,6 +326,91 @@ def test_private_endpoint_connection_storage_account(self, storage_account):
 
         self.cmd('network private-endpoint-connection delete --id {sa_pec_id} -y')
 
+class NetworkPrivateLinkAMLScenarioTest(ScenarioTest):
+    @ResourceGroupPreparer(name_prefix='cli_test_aml_plr')
+    def test_private_link_resource_aml(self):
+        self.kwargs.update({
+            'workspace_name': self.create_random_name('testaml', 20)
+        })
+        result = self.cmd('ml workspace create --workspace-name {workspace_name} --resource-group {rg}').get_output_in_json()
```

Comment:
Does `az ml workspace` command released or not? If it's in extension then you should add `az extension add -n {extension Name}` before test. And you also need to add `@live_only` for those tests.